.github/workflows/update-licenses.yml

```diff
@@ -0,0 +1,33 @@
+name: Update dependabot PR Licenses
```

Comment:
I would love to see each workflow action have a small paragraph in https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/docs/dev/ci.md.

How about we start with this one?

Reason: This does not include any comment what it does and why. Reasoning about the existence of this workflow can be somehow reverse engineered by looking at the `name` and the implementation of `make recompute-licenses` but it might be hard for a future reader.

Reply:
Done